model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -797,6 +798,9 @@ async def create_vllm_bundle(
         if "llama-3-70b" in model_name:
             subcommands[-1] = subcommands[-1] + " --gpu-memory-utilization 0.95 --enforce-eager"
 
+        if "gemma-2" in model_name:
```

Comment:
slightly weird we don't use `infer_addition_engine_args_from_model_name` here, and it looks like they do slightly different things, i.e. for gpu mem util matching llama-3-70b exactly or anything with >=70b params, but I guess that was the case to begin with

Reply:
Agreed; I think there is room for consolidation/improvement but don't want to make too many changes in this PR